arch/arm/core/aarch32/cortex_a_r/Kconfig

```diff
@@ -61,6 +61,12 @@ config GEN_ISR_TABLES
 config GEN_IRQ_VECTOR_TABLE
 	default n
 
+config DCACHE_LINE_SIZE
+	default 32 if CACHE_MANAGEMENT
```

Comment:
I don't think having `if CACHE_MANAGEMENT` is useful here.

The `DCACHE_LINE_SIZE` symbol already has `depends on CACHE_MANAGEMENT` so that's redundant.

Reply:
ok, fair point